tutorials/fundamentals/regexp-starts-with.md

```diff
@@ -0,0 +1,15 @@
+To check the beginning of a string using a regular expression in JavaScript, use the `test()` function.
```

Comment:
"and a regular expression that starts with `^`. The `^` character is a special character that represents the start of the string"